tests/unit/PackagesProvider/ComposerInstalledJsonPackagesProviderTest.php

```diff
@@ -32,7 +32,16 @@ protected function tearDown(): void
     }
 
     /**
-     * @return iterable<array{array<mixed>, Packages, bool}>
+     * @return iterable<array{
```

Comment:
Would it make sense to define (and use?) some of those as phpstan-type in ComposerInstalledJsonPackagesProvider class and just import here? :thinking:

Reply:
Yeah absolutely

Reply:
Done in https://github.com/Lendable/composer-license-checker/pull/65/commits/dbb54a8332d1ec8032c78df66e97afbbf89da515